README.md

```diff
@@ -111,6 +111,11 @@ print(IO::Blob:D: *@text) returns Bool
 
 Text writing; writes the given `@text` to the Blob.
 
+say(IO::Blob:D: *@text) returns Bool
+------------------------------------
+
+Text writing; similar to write, but add a newline to end of the `@text`.
```

Comment:
Oops, you're right: https://github.com/moznion/p6-IO-Blob/pull/13/commits/7089988a60de844e10e106e49e60f323f9fd9ccd